amd_openvx_extensions/amd_rpp/CMakeLists.txt

```diff
@@ -115,7 +115,7 @@ list(APPEND SOURCES
         source/image/RotatebatchPD.cpp
         source/image/SaturationbatchPD.cpp
         source/image/ScalebatchPD.cpp
-        source/image/SequenceRearrange.cpp
+        source/image/SequenceRearrangeImage.cpp
```

Comment:
@fiona-gladwin Why is this "image" others "batchPD"?

Reply:
We named it since this Node does not call any RPP batchPD functions
Changed name to SequenceRearrangebatchPD to maintain uniformity